it_zwierzak/user/admin.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
jeśli mamy pusty plik, to optowałbym za jego usunięciem, wtedy PR wygląda bardziej elegancko